### PR TITLE
🚚 Tidy directory and file names

### PR DESCRIPTION
**Pull request by sschleemilch**
Made directory and filenames more concise. Basically removed `vss` where the context is clear anyway aswell as the `vspec` intermediary directory that did not contain anything else

- `vssexporters` -> `exporters`
- `vss2x.py` -> `x.py`
- `vss_tools/vspec/...` -> `vss_tools/...`

Before:
```
src
└── vss_tools
   ├── __init__.py
   └── vspec
      ├── __init__.py
      ├── cli.py
      ├── cli_options.py
      ├── datatypes.py
      ├── lazy_group.py
      ├── main.py
      ├── model.py
      ├── py.typed
      ├── tree.py
      ├── units_quantities.py
      ├── utils
      │  ├── __init__.py
      │  ├── idgen_utils.py
      │  ├── misc.py
      │  └── vss2id_val.py
      ├── vspec.py
      └── vssexporters
         ├── __init__.py
         ├── vss2apigear.py
         ├── vss2binary.py
         ├── vss2csv.py
         ├── vss2ddsidl.py
         ├── vss2franca.py
         ├── vss2go.py
         ├── vss2graphql.py
         ├── vss2id.py
         ├── vss2json.py
         ├── vss2jsonschema.py
         ├── vss2protobuf.py
         ├── vss2samm
         │  ├── config
         │  │  └── config.py
         │  ├── helpers
         │  │  ├── data_types_and_units.py
         │  │  ├── file_helper.py
         │  │  ├── namespaces.py
         │  │  ├── samm_concepts.py
         │  │  ├── string_helper.py
         │  │  ├── ttl_builder_helper.py
         │  │  ├── ttl_helper.py
         │  │  └── vss_helper.py
         │  └── vss2samm.py
         ├── vss2tree.py
         └── vss2yaml.py

```

Now:
```
src
└── vss_tools
   ├── __init__.py
   ├── cli.py
   ├── cli_options.py
   ├── datatypes.py
   ├── exporters
   │  ├── __init__.py
   │  ├── apigear.py
   │  ├── binary.py
   │  ├── csv.py
   │  ├── ddsidl.py
   │  ├── franca.py
   │  ├── go.py
   │  ├── graphql.py
   │  ├── id.py
   │  ├── json.py
   │  ├── jsonschema.py
   │  ├── protobuf.py
   │  ├── samm
   │  │  ├── __init__.py
   │  │  ├── config
   │  │  │  └── config.py
   │  │  └── helpers
   │  │     ├── data_types_and_units.py
   │  │     ├── file_helper.py
   │  │     ├── namespaces.py
   │  │     ├── samm_concepts.py
   │  │     ├── string_helper.py
   │  │     ├── ttl_builder_helper.py
   │  │     ├── ttl_helper.py
   │  │     └── vss_helper.py
   │  ├── tree.py
   │  └── yaml.py
   ├── lazy_group.py
   ├── main.py
   ├── model.py
   ├── py.typed
   ├── tree.py
   ├── units_quantities.py
   ├── utils
   │  ├── __init__.py
   │  ├── idgen_utils.py
   │  ├── misc.py
   │  └── vss2id_val.py
   └── vspec.py

```

### CODE REVIEW

File: .github/workflows/buildcheck.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
           poetry run flake8 src tests contrib
       - name: Run tests
         run: |
-          poetry run pytest --cov=vss_tools --cov-report=term-missing --cov-fail-under=90
+          poetry run pytest --cov=vss_tools --cov-report=term-missing --cov-fail-under=88
```
Comment (SebastianSchildt) [inline .github/workflows/buildcheck.yml:48]: Are you sneaking in quality reductions here :D ?

Comment (sschleemilch) [reply .github/workflows/buildcheck.yml:48]: Kind of... I have no clue why it dropped with that change to be honest

**Comment by erikbosch** [comment]
MoM:

- Please review
- Merge decision next week

**Review by erikbosch — APPROVED** [review]
LGTM

But I think leaving it open until end of week makes sense to give others a chance to review

**Pull request merged**